### ISSUE TITLE
taxonomyFilter not working with where variable 

### ISSUE THREAD

**Issue by mehidi258** [wp-graphql/wp-graphql-woocommerce#460]
Here is my query 
query PRODUCTS_FILTER_QUERY($where: RootQueryToProductConnectionWhereArgs!) {
  products(where: $where) {
    edges {
      node {
        name
      }
    }
  }
}

here is my variables {
  "where": {
    "taxonomyFilter": {
      "and": {"taxonomy": "PACOLOR", "terms":"green"} 
    }
  }
}

It's giving this error 
"Variable \"$where\" got invalid value {\"taxonomyFilter\":{\"and\":{\"taxonomy\":\"PACOLOR\",\"terms\":\"green\"}}}; Field \"taxonomy\" is not defined by type ProductTaxonomyFilterRelationInput at value.taxonomyFilter.and."

Is there any way to handle it?
Thanks!

**Comment by niklaslovgren** [comment]
Instead of:
```json
{
   "taxonomyFilter": {
      "and": {
         "taxonomy": "PACOLOR",
         "terms": "green"
      }
   }
}
```

Try this:
```json
{
   "taxonomyFilter": [
      {
         "and": [
            {
               "taxonomy": "PACOLOR",
               "terms": [
                  "green"
               ]
            }
         ]
      }
   ]
}
```

**Comment by kidunot89** [comment]
@mehidi258 @niklaslovgren I corrected the shape of the `taxonomyFilter` in #495. See the PR summary for examples of it's use. It should work perfect with most other whereArgs.
